code/datums/mutations.dm

```diff
@@ -531,6 +531,10 @@
 /datum/mutation/human/swedish/say_mod(message)
 	if(message)
 		message = replacetext(message,"w","v")
+		message = replacetext(message,"j","y")
+		message = replacetext(message,"a",pick("�","�","�","a"))
+		message = replacetext(message,"bo","bjo")
+		message = replacetext(message,"o",pick("�","�","o"))
```

Comment:
byond doesn't support formats outside of the western iso encodings afaik. These will probably get mangled depending on the users locale settings.